# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
# do not restrict libmcfp to version 1.2.4
diff -ru mrc-1.3.13/CMakeLists.txt mrc-1.3.13_update_libmcfp134/CMakeLists.txt
--- mrc-1.3.13/CMakeLists.txt	2024-07-22 10:27:30.000000000 +0200
+++ mrc-1.3.13_update_libmcfp134/CMakeLists.txt	2025-01-14 13:40:27.793832892 +0100
@@ -61,7 +61,7 @@
 	message(STATUS "Using COFF spec: ${COFF_SPEC}")
 endif()
 
-find_package(libmcfp 1.2.4 QUIET)
+find_package(libmcfp CONFIG REQUIRED)
 
 if(NOT libmcfp_FOUND)
 	include(FetchContent)
